server/api/core/templates.js

```diff
@@ -3,7 +3,8 @@ import ReactDOMServer from "react-dom/server";
 import Handlebars from "handlebars";
 import Immutable from "immutable";
 import { Import } from "./import";
-import { Templates } from "/lib/collections";
+import { Assets, Shops, Templates } from "/lib/collections";
```

Comment:
I'm seeing linting errors, `Import`, `Shops` not used.

`export function Templates() {` already defined.